oe-init-build-env

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
+DIR=$( dirname -- "${BASH_SOURCE[0]}" )
+TEMPLATECONF=${DIR}/meta-lxatac-software/conf/templates/default/
```

Comment:
Hi,

our first external contribution, exciting!

The previous solution using `pwd` made sure that `$TEMPLATECONF` is an absolute path.
Now `$TEMPLATECONF` can be a relative path (e.g. `./`). This results in the script failing to source (at least for me):

```
lgo@dude03:~/work/Projects/meta-lxatac$ source oe-init-build-env
Error: TEMPLATECONF value points to nonexistent directory './meta-lxatac-software/conf/templates/default/'
```

I would suggest taking inspiration from `poky/oe-init-build-env` and adding a `readlink -f`:

https://github.com/yoctoproject/poky/blob/5614e6b38a55df5df95556853961610c2b6bf329/oe-init-build-env#L38-L42

```suggestion
DIR=$( dirname -- "${BASH_SOURCE[0]}" )
DIR=$( readlink -f "${DIR}" )

TEMPLATECONF=${DIR}/meta-lxatac-software/conf/templates/default/
```

Reply:
Not a big deal. I integrated your suggestion into the commit and force-pushed.